Improve public endpoints documentation (#158)
etherlinkcom/docs: docs/get-started/network-information.mdx

For current and historical status information for Etherlink, see https://status.etherlink.com.

For every network, we provide two endpoints: a public RPC endpoint and a relay
endpoint. The public RPC endpoint can be consumed by Etherlink users and
decentralized applications to interact with the chain. The relay endpoint
exposes a restricted RPC API with only `eth_sendRawTransaction` and

> `eth_blockNumber` available. The relay endpoint should be preferred when
> setting up an [Etherlink node](/network/evm-nodes).

Reviewer: ```suggestion
`eth_blockNumber` available. In most cases, use the relay endpoint when
setting up an [Etherlink node](/network/evm-nodes).
```

Author: Maybe we can drop “In most cases” even?

Reviewer: Sure, unless there are cases where you would need to use the public endpoint. "Public endpoint" isn't a great name, though, because both endpoints are publicly accessible. Maybe "full RPC endpoint?"

Author: I’ve tried something, let me know what you think!

Reviewer: Yeah, I think that's clear.

> :::warning Public RPC Endpoint Restrictions

Reviewer: Should we also mention these limitations (batch limit and supported endpoints) on the "Ethereum endpoint support" page?

Author: I’m not sure, it’s not directly related to the software we distribute and deploying your own node allow to not have them.

But no strong opinion either.
